src/libs/SidebarUtils.ts

```diff
@@ -447,8 +455,60 @@ function getOptionData({
 
     return result;
 }
+function getReportBeginningOfChatHistoryMessage(report: OnyxEntry<Report>): string {
+    if (ReportUtils.isThread(report)) {
+        return Localize.translateLocal('report.noActivityYet');
+    }
+
+    const welcomeMessage = ReportUtils.getWelcomeMessage(report);
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (report?.description) {
+            return parseHtmlToText(report.description);
+        }
+        return `${welcomeMessage.phrase1} ${ReportUtils.getDisplayNameForParticipant(report?.ownerAccountID)} ${welcomeMessage.phrase2} ${ReportUtils.getPolicyName(report)} ${
+            welcomeMessage.phrase3
+        }`;
+    }
+
+    if (ReportUtils.isChatRoom(report)) {
+        if (report?.description) {
+            return parseHtmlToText(report.description);
+        }
+        return `${welcomeMessage.phrase1} ${welcomeMessage.showReportName ? ReportUtils.getReportName(report) : ''} ${welcomeMessage.phrase2 ?? ''}`;
+    }
+
+    if (ReportUtils.isSelfDM(report) || ReportUtils.isSystemChat(report)) {
+        return `${welcomeMessage.phrase1}`;
+    }
+
+    const participantAccountIDs = ReportUtils.getParticipantsAccountIDsForDisplay(report);
+    const isMultipleParticipant = participantAccountIDs.length > 1;
+    const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
+        OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, allPersonalDetails),
+        isMultipleParticipant,
+    );
+    const displayNamesWithTooltipsText = displayNamesWithTooltips
+        .map(({displayName, pronouns}, index) => {
+            const formattedText = !pronouns ? displayName : `${displayName} (${pronouns})`;
```

Comment:
Is there a reason that we are not simply using `displayName` as in `ReportWelcomeText`?

Reply:
this logic is added from here 
 https://github.com/Expensify/App/blob/83eb7e6ee5f8a86c4500deda3454763680dcd5cf/src/libs/SidebarUtils.ts#L399